### DISCUSSION TITLE
蔡徐坤本人正式入驻绳网

### DISCUSSION THREAD

**Discussion by Yang66g** [share121/inter-knot#1615] [其他]
![image](https://github.com/user-attachments/assets/f067b629-0c3a-40d4-ae08-667f70d26b93)
![Uploading image.png…]()
听说绳网环境很好，应该不会有小黑子吧

**Comment by LyxB-STRRRR** [comment]
姬妮钛镁

**Comment by Git-ys1** [comment]
泥干吗矮油
